backend/site-api/src/app/admin/reset_all.rs

```diff
@@ -48,6 +48,10 @@ pub async fn reset_all(
     // Clear all caches
     context.cache.cache.invalidate_all();
     context.cache.sync().await;
+    context.cache.metadata_cache.invalidate_all();
+    context.cache.metadata_cache.run_pending_tasks().await;
+    context.cache.domain_cache.invalidate_all();
+    context.cache.domain_cache.run_pending_tasks().await;
```

Comment:
I noticed some tests failing periodically after my changes, I'm not sure if they failed before but this fixes it!

Reply:
Oh, I think that's the reason! Sorry, I forgot to add the relevant code here.